files/zh-cn/web/api/selection/rangecount/index.md

```diff
@@ -26,12 +26,21 @@ value = sel.rangeCount
 ### HTML Content
 
 ```html
-//打开控制台看看 selection 中有多少 range 对象。
-//在 Gecko 浏览器，当你用鼠标在表格单元格中拖动的同时按住 Ctrl，你可以选择多个 range。
+//打开控制台看看 selection 中有多少 range 对象。 //在 Gecko
+浏览器，当你用鼠标在表格单元格中拖动的同时按住 Ctrl，你可以选择多个 range。
```

Comment:
I'd like to update the example

https://developer.mozilla.org/en-US/docs/Web/API/Selection/rangeCount